src/translation/translation_fr_FR.ts

```diff
@@ -3585,12 +3590,12 @@ Nous avons coupé votre canal et activé &quot;Me silencer&quot;. Veuillez d&apo
     <message>
         <location filename="../util.h" line="583"/>
         <source>None</source>
-        <translation type="unfinished">Aucune</translation>
+        <translation>Aucune</translation>
     </message>
     <message>
         <location filename="../util.h" line="630"/>
         <source>Not registered</source>
-        <translation type="unfinished"></translation>
+        <translation>Non inscrit</translation>
     </message>
     <message>
         <location filename="../util.h" line="633"/>
```

Comment:
```
    <message>
        <location filename="../util.h" line="648"/>
        <source>Directory server list full</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Directory Server full</source>
        <translation type="vanished">Serveur annuaire plein</translation>
    </message>
```
These need resolving - the English is correct in the first ("unfinished") entry and the French correct in the second ("vanished") entry.

Reply:
done